net/dpdk/Makefile

```diff
@@ -0,0 +1,41 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME := dpdk
+PKG_VERSION := 21.05
+
+PKG_SOURCE := dpdk-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL := http://static.dpdk.org/rel
+PKG_HASH := 1e12499b4c5fcdb57c83e5fe184ea6becddf7cf092893c2c5ef2efb0eec12f0b
+
+PKG_BUILD_DEPENDS:= meson/host ninja/host
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/host-build.mk
+include $(TOPDIR)/feeds/packages/devel/meson/meson.mk
+
+TARGET_CFLAGS += -D_GNU_SOURCE -Wno-unused-result -Wno-format-nonliteral
+
+define Package/dpdk
+  SECTION    := net
+  CATEGORY   := Network
+  TITLE      := Data Plane Development Kit (DPDK)
+  DEPENDS    := @(x86_64||arm||aarch64) +librt +libpthread +libbpf +libpcap +libopenssl
+endef
+
+define Package/dpdk/description
+  DPDK consists of libraries to accelerate packet processing
+  workloads running on a wide variety of CPU architectures.
+endef
+
+define Package/dpdk/install
+	$(INSTALL_DIR) $(1)/usr/bin
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/dpdk-testpmd $(1)/usr/bin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/dpdk-proc-info $(1)/usr/bin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/dpdk-test-flow-perf $(1)/usr/bin/
```

Comment:
Arguably, these 3 as well into a single command...

Reply:
Then I've been doing it wrong... :grin:

Seriously, I see examples of both:

https://github.com/openwrt/packages/blob/master/net/strongswan/Makefile#L580

People put a lot of effort into getting the builds to run faster, use more caching, more parallelism, etc.

A cheap low-hanging fruit we could leverage more is using a single command in the place of _N_ instances of the same command (`mkdir`, `cp`, `install`, etc.) each with a single argument.

Reply:
@pprindeville ..Addressed review comment